examples/advanced_operations/add_ad_group_asset.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
```

Comment:
This should go in the misc subdirectory until we decide how to group asset operations.

Rename file to add_ad_group_image_asset.py

Reply:
I've added to a new file to a misc and removed  advanced_operations.